x11-themes/mate-backgrounds/mate-backgrounds-1.27.1.ebuild

```diff
@@ -5,8 +5,9 @@ EAPI=8
 
 inherit mate
 
-if [[ ${PV} != 9999 ]]; then
-	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+MINOR=$((ver_cut 2) % 2)
+if [[ ${MINOR} -eq 0 ]]; then
+        KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
```

Comment:
This seems a bit weird but let's fix the tree first!

Reply:
It's actually not very weird - I think we have other examples somewhere. Even releases are stable, odd are development.